orcoursetrion/lib/github.py

```diff
@@ -64,6 +66,38 @@ def _get_all(self, url):
                 results += response.json()
         return results
 
+    def _find_team(self, org, team):
+        """Find a team in an org by name, or raise.
+
+        Args:
+            org (str): Organization to create the repo in.
+            team (str): Team to find by name.
+
+        Raises:
+            GitHubUnknownError
+            GitHubNoTeamFound
+
+        Returns:
+            dict: Team dictionary
+                  (https://developer.github.com/v3/orgs/teams/#response)
+        """
+        list_teams_url = '{url}orgs/{org}/teams'.format(
+            url=self.api_url,
+            org=org
+        )
+        teams = self._get_all(list_teams_url)
+        if not teams:
+            raise GitHubUnknownError(
+                'No teams found in {0} organization'.format(org)
+            )
```

Comment:
So I didn't want to have to check for a 404, vs a 422, vs 500, and there shouldn't ever be a situation where an org doesn't have a team since it always has `Owners`.  So no teams coming back really is an unknownerror. I could maybe move some status code checking up into _get_all so that I can at least dump out the response from github with the error.

I'll work on that.